### PR TITLE
Concourse & Artifactory

### PR DESCRIPTION
**Pull request by track0x1**
Adding docs for setting up / using concourse & artifactory

### CODE REVIEW

File: references/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# References
+
+## Table of Contents
```
Comment (sidforhbc) [inline references/README.md]: Move this up to the main README's table of contents.

File: references/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Artifactory
+
+> We use [Artifactory](https://www.jfrog.com/artifactory/) to store all our internal NodeJS packages. This is equivalent to using [npm Enterprise](https://www.npmjs.com/enterprise).
+
+### Setup with NPM
+
+To access the [Artifactory web interface](https://hbc.jfrog.io/hbc/webapp/), you will need to authenticate through [Okta](https://hbctech.okta.com/).
+
+If you would like to easily switch between npm repositories (i.e. default, work), we suggest using [npmrc](https://github.com/deoxxa/npmrc).
```
Comment (sidforhbc) [inline references/README.md]: I wouldn't even bother with this; it adds a level of complexity that we don't need to support as an organization

File: references/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+"publishConfig": {
+  "registry":"https://hbc.jfrog.io/hbc/api/npm/hbc-virt-npm"
+}
+```
+
+## Concourse
+
+> Concourse is the continuous integration / continuous deployment (CI/CD) we use at HBC Tech. [Learn more...](http://concourse.ci)
+
+### Installing `fly`
```
Comment (sidforhbc) [inline references/README.md]: Let's call this "Setting up Concourse" and then add a section for "Using Concourse"

File: references/README.md (hunk 1 above)
Comment (track0x1) [reply references/README.md]: I wanted to keep the main README super minimal so people can hopefully get what they need fast. I can imagine this references page will grow quite large given all of common questions people have.

**Pull request merged**